doc/source/user_guide/missing_data.rst

```diff
@@ -455,7 +462,7 @@ at the new values.
    ser = pd.Series(np.sort(np.random.uniform(size=100)))
 
    # interpolate at new_index
-   new_index = ser.index | pd.Index([49.25, 49.5, 49.75, 50.25, 50.5, 50.75])
+   new_index = pd.Float64Index(ser.index | pd.Index([49.25, 49.5, 49.75, 50.25, 50.5, 50.75]))
```

Comment:
I would leave this change out. 

Yes, it is annoying that it fails, but this should work and is a regression we need to fix (and we actually catched this due to the docs). 

If this is the final error that is remaining and blocking that we can start failing CI for errors/warnings in the docs, then we can do that or add a temporary `:okexcept:`, but we are not there yet I think.